neurodamus/utils/memory.py

```diff
@@ -99,3 +99,27 @@ def get_mem_usage():
     usage_mb = float(data_size) * os.sysconf("SC_PAGE_SIZE") / 1024 ** 2
 
     return usage_mb
+
+
+class SynapseMemoryUsage:
+    ''' A small class that works as a lookup table
+    for the memory used by each type of synapse.
+    The values are in KB. The values cannot be set by the user.
+    '''
+    def __init__(self):
+        self._synapse_memory_usage = {
```

Comment:
Since `_synapse_memory_usage` is constant I would promote to class_level. Then we can drop `__init__` and make `get_memory_usage` a classmethod to avoid requiring instances, which in this case makes little sense.

Reply:
Thanks for the clarification. Fixed!